web/clusters.go

```diff
@@ -261,16 +263,183 @@ func (nodes Nodes) PassingCount() int {
 	return warning
 }
 
+type ClustersRow struct {
+	Id              string
+	Name            string
+	Health          string
+	SIDs            []string
+	Type            string
+	ResourcesNumber int
+	HostsNumber     int
+}
+
+type ClustersTable []*ClustersRow
+
+func NewClustersTable(clusters map[string]*cluster.Cluster, hosts map[string]hosts.HostList) ClustersTable {
+	var clusterTable ClustersTable
+
+	for id, c := range clusters {
+		var health string
+		var sids []string
+
+		hl, ok := hosts[id]
+
+		if ok && len(hl) > 0 {
+			health = hl.Health()
+			set := make(map[string]struct{})
+
+			for _, h := range hl {
+				ss, err := h.GetSAPSystems()
```

Comment:
I don't really think we should do this `GetSAPSystems`. We should get the SID from the cluster resources (it should be in some cluster node attribute).
In fact, I'm not sure if the cluster should have multiple sids. It's true that the cost-optimized manages 2 sids though

Reply:
to follow up in #143